### DISCUSSION TITLE
翻譯歧異討論

### DISCUSSION THREAD

**Discussion by Liteya** [GpointChen/FFXIVChnTextPatch-GP#369] [General]
![5618AS](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/101965725/4a9f0a71-ba16-48cf-9211-55ad58eaceec)
日文原文「重曹」，簡中翻譯是「泡碱」，繁中應為「泡鹼」。鹼性的鹼。
截圖中的翻譯為「泡堿」。「堿」同「㙳」，表示地上凹陷之處。
有人回報這個翻譯的歧異 GpointChen 大大 思考一下 要不要改這個部分?

**Comment by GpointChen** [comment]
Ok，預計之後更新時一併修正
